### PR TITLE
Adds documentation on usage of the PostgresChannelMessageTableSubscriber.

### PR DESCRIPTION
**Pull request by raphw**
Adds a segment on the newly added class in the documentation and refers to it in the section on using databases as a queue.

### CODE REVIEW

File: src/reference/asciidoc/jdbc.adoc [hunk 1]
```diff
@@ -480,6 +480,9 @@ Instead, if possible, consider using either JMS- or AMQP-backed channels instead
 For further reference, see the following resource:
 
 * https://mikehadlow.blogspot.com/2012/04/database-as-queue-anti-pattern.html[The Database As Queue Anti-Pattern].
+
+If you are still planning to use your database as a queue, consider using PostgreSQL and its notification
+mechanism which is described in a subsequent section.
```
Comment (artembilan) [inline src/reference/asciidoc/jdbc.adoc:485]: Great! Can you, please, follow the rule of `One sentence per line`: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line ?

File: src/reference/asciidoc/jdbc.adoc [hunk 2]
```diff
@@ -575,6 +578,29 @@ The message data for a persistent channel is keyed in the store on the channel n
 Consequently, if the channel names are not globally unique, the channels can pick up data that is not intended for them.
 To avoid this danger, you can use the message store `region` to keep data separate for different physical channels that have the same logical name.
 
+==== PostgreSQL: receiving push notifications
```
Comment (artembilan) [inline src/reference/asciidoc/jdbc.adoc:581]: Add, please, an `id` to this section, e.g. `[[postgres-notifications]]`.
And then add a new section in the `whats-new.adoc` in the `New Components` paragraph.

File: src/reference/asciidoc/jdbc.adoc [hunk 3]
```diff
@@ -575,6 +578,29 @@ The message data for a persistent channel is keyed in the store on the channel n
...
+PostgreSQL offers a listen and notification framework for receiving push notifications upon database table manipulations.
+Spring Integration leverages this mechanism to allow for receiving push notifications when new messages are added to a
+`JdbcChannelMessageStore`. When using this feature, a database trigger must be defined, which can be found as part of the
+comments of the `schema-postgresql.sql` file which is included in the JDBC module of Spring Integration.
+
+Push notifications are received via the `PostgresChannelMessageTableSubscriber` class which allows its subscribers to
+receive a callback upon the arrival of new messages for any given `region` and `groupId`. These notifications are received
+even if a message was appended on a different JVM, but to the same database. To read messages when they arrive, a
+`PostgresSubscribableChannel` can delegate new messages to a `MessageHandler` what allows to implement an effective
+pub-sub mechanism based on JDBC.
```
Comment (artembilan) [inline src/reference/asciidoc/jdbc.adoc:592]: I guess would be great to add a simple sample of the possible configuration of the channel and that `PostgresChannelMessageTableSubscriber` .
As `@Bean`s, of course! 😄

**Review by artembilan — CHANGES_REQUESTED** [review]

**Comment by raphw** [comment]
Not sure how I can validate the formatting locally, but I addressed your comments.

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration/commit/145edb238e1764880faa7f58d6577a91ac9b2e8f.

Thank you for prompt turn around!

**Pull request closed without merging**